lib/cc/cli/path_fingerprint.rb

```diff
@@ -0,0 +1,33 @@
+require "digest/md5"
+
+module CC
```

Comment:
Nit: there is a lot of hash access going on in these two fingerprint classes. Would it be nicer in the long run to add more accessors to `Issue`?